<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicalSegmentationDlg</class>
 <widget class="QDialog" name="GraphicalSegmentationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Segmentation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="buttonsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="pauseButton">
        <property name="toolTip">
         <string>Pause segmentation (Space)</string>
        </property>
        <property name="statusTip">
         <string>Pause segmentation  (allow rotation/panning of 3D view)</string>
        </property>
        <property name="text">
         <string>pause</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="selectionModelButton">
        <property name="toolTip">
         <string>Polyline selection mode (Tab)</string>
        </property>
        <property name="text">
         <string>polyline selection</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/polygonSelect.png</normaloff>:/images/polygonSelect.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="inButton">
        <property name="toolTip">
         <string>Segment In (I)</string>
        </property>
        <property name="statusTip">
         <string>Segment (keep points inside)</string>
        </property>
        <property name="text">
         <string>in</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/segmentIn.png</normaloff>:/images/segmentIn.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="outButton">
        <property name="toolTip">
         <string>Segment Out (O)</string>
        </property>
        <property name="statusTip">
         <string>Segment (keep points outside)</string>
        </property>
        <property name="text">
         <string>out</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/segmentOut.png</normaloff>:/images/segmentOut.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="undoButton">
        <property name="toolTip">
         <string>undo (Ctrl+Z)</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="redoButton">
        <property name="toolTip">
         <string>redo (Ctrl+Y)</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="resetButton">
        <property name="toolTip">
         <string>reset</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/reset.png</normaloff>:/images/reset.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="validButton">
        <property name="toolTip">
         <string>Confirm segmentation (Enter)</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/validate.png</normaloff>:/images/validate.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cancelButton">
        <property name="toolTip">
         <string>Cancel segentation (Esc)</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionSetRectangularSelection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/rectangleSelect.png</normaloff>:/images/rectangleSelect.png</iconset>
   </property>
   <property name="text">
    <string>Rectangular selection</string>
   </property>
   <property name="toolTip">
    <string>Activate rectangular selection</string>
   </property>
  </action>
  <action name="actionSetPolylineSelection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/polygonSelect.png</normaloff>:/images/polygonSelect.png</iconset>
   </property>
   <property name="text">
    <string>Polygonal selection</string>
   </property>
   <property name="toolTip">
    <string>Activate polyline selection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
